src/IndexerBase.php

```diff
@@ -55,7 +55,8 @@ public function index(array $document) {
 
     $this->database->query('
       INSERT IGNORE INTO {kifisearch_index} (entity_id, entity_type)
```

Comment:
What do you think about removing the IGNORE from the insert with this change? To me it seems like it may not be useful anymore and could obscure other issues in the inserts.

Reply:
Yeah, it kinda does not anything in the current version. I removed the IGNORE statement.